pkg/network/vmispec/interface.go

```diff
@@ -52,6 +52,18 @@ func FilterInterfacesSpec(ifaces []v1.Interface, predicate func(i v1.Interface)
 	return filteredIfaces
 }
 
+func IsMigratableBindingPlugin(networks []v1.Network, ifaces []v1.Interface, bindingPlugins map[string]v1.InterfaceBindingPlugin) bool {
```

Comment:
Looking at the usage, I think we need a general check about the migration support for any binding/interface.

I do not think there is a need to travel the codebase to see what is considered migratable and what is not, once place is preferred.

Then, the logic will be simple:
- Lookup for the pod network & interface.
- If it falls under the supported cases (masquerade, bridge with annotation and binding plugin with the flag set), then it returns `true`.